lib/deck.rb

```diff
@@ -0,0 +1,17 @@
+class Deck
+attr_reader :cards
+  def initialize(cards)
+    @cards = cards
+
+  end
+    def count
+     @cards.length
+    end
```

Comment:
this code is about 1 indent too far, have it line up with the last method, and have one line in between each method.